app/main/views.py

```diff
@@ -99,6 +165,43 @@ def get_service(service_id):
     return jsonify(services=jsonify_service(service))
 
 
+@main.route('/services-archive/<int:service_archive_id>', methods=['GET'])
```

Comment:
`services-archive` doesn't seem like the right name for the resource. We chose `/services` because the resource we're dealing with is `service` and we decided to go with plurals for our resource endpoints. Using the same train of thought for this; the individual resource we're talking about is a service archive so the resource endpoint should be `service-archives`.